scylla.go

```diff
@@ -181,6 +207,13 @@ type scyllaConnPicker struct {
 	nrShards    int
 	msbIgnore   uint64
 	pos         uint64
+
```

Comment:
This var grouping is weird why do we need 3 groups?

Reply:
I put new variables with the old variables so that there is only one group.